redfish_client/resource.py

```diff
@@ -67,6 +71,10 @@ def _build_from_hash(self, data):
         else:
             return Resource(self._connector, data=data)
 
+    def _clear_cache(self):
```

Comment:
This function is called "clear cache" but in fact what it does is "refresh cache" therefore I'd rename it into `_refresh_cache`